internal/commands/servergroup/show.go

```diff
@@ -0,0 +1,106 @@
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/output"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/resolver"
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/ui"
+
+	"github.com/UpCloudLtd/upcloud-go-api/v6/upcloud/request"
+)
+
+// ShowCommand creates the "servergroup show" command
+func ShowCommand() commands.Command {
+	return &showCommand{
+		BaseCommand: commands.New(
+			"show",
+			"Show server group details",
+			"upctl servergroup show 8abc8009-4325-4b23-4321-b1232cd81231",
+			"upctl servergroup show my-server-group",
+		),
+	}
+}
+
+type showCommand struct {
+	*commands.BaseCommand
+	resolver.CachingServerGroup
+	completion.ServerGroup
+}
+
+// Execute implements commands.MultipleArgumentCommand
+func (s *showCommand) Execute(exec commands.Executor, uuid string) (output.Output, error) {
+	svc := exec.All()
+	serverGroup, err := svc.GetServerGroup(exec.Context(), &request.GetServerGroupRequest{UUID: uuid})
+	if err != nil {
+		return nil, err
+	}
+
+	antiAffinityStatus := output.Combined{}
+
+	for i, memberStatus := range serverGroup.AntiAffinityStatus {
+		var serverTitle, serverHostname, serverHost, serverZone string
+		if serverDetails, err := svc.GetServerDetails(exec.Context(), &request.GetServerDetailsRequest{UUID: memberStatus.ServerUUID}); err != nil {
+			serverTitle = ""
+			serverHostname = ""
+			serverHost = ""
+			serverZone = ""
+		} else {
+			serverTitle = serverDetails.Title
+			serverHostname = serverDetails.Hostname
+			serverHost = strconv.Itoa(serverDetails.Host)
+			serverZone = serverDetails.Zone
+		}
+
+		antiAffinityStatus = append(antiAffinityStatus, output.CombinedSection{
+			Contents: output.Combined{
+				output.CombinedSection{
+					Contents: output.Details{
+						Sections: []output.DetailSection{
+							{
+								Title: fmt.Sprintf("Server %d (%s):", i+1, memberStatus.ServerUUID),
+								Rows: []output.DetailRow{
+									{Title: "Title:", Value: serverTitle},
+									{Title: "Hostname:", Value: serverHostname},
+									{Title: "Host:", Value: serverHost},
+									{Title: "Zone:", Value: serverZone},
+									{Title: "Anti-affinity state:", Value: string(memberStatus.Status), Format: format.ServerGroupAntiAffinityState},
+								},
+							},
+						},
+					},
+				},
+			},
+		})
+	}
+
+	// For JSON and YAML output, passthrough API response
+	return output.MarshaledWithHumanOutput{
+		Value: serverGroup,
+		Output: output.Combined{
+			output.CombinedSection{
+				Contents: output.Details{
+					Sections: []output.DetailSection{
+						{
+							Title: "Overview:",
+							Rows: []output.DetailRow{
+								{Title: "UUID:", Value: serverGroup.UUID, Colour: ui.DefaultUUUIDColours},
+								{Title: "Title:", Value: serverGroup.Title},
+								{Title: "Anti-affinity policy:", Value: serverGroup.AntiAffinityPolicy},
+							},
+						},
+					},
+				},
+			},
+			labels.GetLabelsSection(serverGroup.Labels),
+			output.CombinedSection{
+				Contents: antiAffinityStatus,
```

Comment:
Could we use similar table here as in server list output, but with plan and status changed into host and AA status? For example:

```txt
UUID                                   Hostname        Zone        Host         Anti-Affinity state 
────────────────────────────────────── ─────────────── ─────────── ──────────── ────────────────────
 007abd4b-ead7-4160-976c-fd1593b44c7b   orange-mk2pk    de-fra1     1234567890   unmet 
 00f5cd98-76c7-4404-8fee-fe4622901d03   orange-nfktc    de-fra1     1234567890   unmet
```

Reply:
Using table now, looks better. Added the combined state and server count to overview as well.